src/ansys/aedt/core/hfss.py

```diff
@@ -5885,7 +5889,7 @@ def get_antenna_data(
         if frequencies is not None:
             if type(frequencies) in [float, int, str]:
                 frequencies = [frequencies]
-            frequencies = [str(freq) + frequency_units for freq in frequencies if is_number(freq)]
+            frequencies = [f"{freq}{frequency_units}" if is_number(freq) else f"{freq}" for freq in frequencies]
```

Comment:
I'll work on a much wider refactoring on frequency units but for the moment the fix is ok for me